datadog/resource_datadog_integration_aws_lambda_arn.go

```diff
@@ -56,9 +53,10 @@ func resourceDatadogIntegrationAwsLambdaArnExists(d *schema.ResourceData, meta i
 	// Exists - This is called to verify a resource still exists. It is called prior to Read,
 	// and lowers the burden of Read to be able to assume the resource exists.
 	providerConf := meta.(*ProviderConfiguration)
-	client := providerConf.CommunityClient
+	datadogClientV1 := providerConf.DatadogClientV1
+	authV1 := providerConf.AuthV1
 
-	logCollections, err := client.GetIntegrationAWSLogCollection()
+	logCollections, _, err := datadogClientV1.AWSLogsIntegrationApi.ListAWSLogsIntegrations(authV1).Execute()
```

Comment:
Should we be checking the httpresp and error code (in addition to the `err` object) and failing if not 2xx? (Same question for all the api calls here)

Reply:
I am updating the PR @nick here is the pr to address the first comment: https://github.com/terraform-providers/terraform-provider-datadog/pull/498

In regards to the second comment none of our resources actively check for the http_response at the moment. I think checking the http resp would unnecessarily complicate the code at the moment. None 2xx responses are marked as errors. What do you think?

Reply:
If needed we will do in another pr

Reply:
My main question here is if there is a case where `err` isn't set but httpresp isn't a 2xx. 
I just want to confirm we're continuing to have the same behavior as before this switch.